next/components/forms/Accordion.tsx

```diff
@@ -0,0 +1,58 @@
+import cx from 'classnames'
+import React, { useState } from 'react'
+
+import ExpandMoreIcon from './icon-components/ExpandMoreIcon'
+
+type AccordionBase = {
+  size: 'sm' | 'md' | 'lg'
+  title: string
+  content: string
+  className?: string
+}
+
+const Accordion = ({ title, content, size }: AccordionBase) => {
+  const [isActive, setIsActive] = useState(false)
+
+  const accordionContainerStyle = cx('flex flex-col gap-4 w-[480px] border-2 border-solid rounded-xl bg-white hover:border-form-calendar-placeholder', {
+    'p-5': size === 'sm',
+    'py-6 px-8': size === 'md',
+    'py-8 px-10': size === 'lg',
+    'h-16': !isActive && size === 'sm',
+    'h-20': !isActive && size === 'md',
+    'h-26': !isActive && size === 'lg',
+    'border-form-input-default': !isActive,
+    'border-form-black-default': isActive,
+  })
+
+  return (
+    <div className={accordionContainerStyle}>
+      <div className={cx('flex items-center justify-between cursor-pointer', {
+        'h-5.5': size === 'sm',
```

Comment:
let's wait for the [spacing PR](https://github.com/bratislava/bratislava.sk/pull/370) to get merged and recheck this afterwards, the 5.5 class won't exist afterwards and perhaps some of the others might be slightly different